test/widgets/action_sheet_test.dart

```diff
@@ -490,4 +491,32 @@ void main() {
       check(await Clipboard.getData('text/plain')).isNull();
     });
   });
+
+  group('CopyMessageLinkButton', () {
+    setUp(() async {
+      TestDefaultBinaryMessengerBinding.instance.defaultBinaryMessenger.setMockMethodCallHandler(
+        SystemChannels.platform,
+        MockClipboard().handleMethodCall,
+      );
+    });
+
+    Future<void> tapCopyMessageLinkButton(WidgetTester tester) async {
+      await tester.ensureVisible(find.byIcon(Icons.link, skipOffstage: false));
+      await tester.tap(find.byIcon(Icons.link));
+      await tester.pump(); // [MenuItemButton.onPressed] called in a post-frame callback: flutter/flutter@e4a39fa2e
+    }
+
+    testWidgets('copies message link to clipboard', (tester) async {
+      final message = eg.streamMessage();
+      final narrow = TopicNarrow.ofMessage(message);
+      await setupToMessageActionSheet(tester, message: message, narrow: narrow);
+
+      final store = await testBinding.globalStore.perAccount(eg.selfAccount.id);
```

Comment:
nit: Same here, I think both blocks are handling the arrangement steps for the test case so that extra white space feels as an extra
```suggestion
      await setupToMessageActionSheet(tester, message: message, narrow: narrow);
      final store = await testBinding.globalStore.perAccount(eg.selfAccount.id);
```